src/js/sdk.tsx

```diff
@@ -138,7 +138,7 @@ export function init(passedOptions: ReactNativeOptions): void {
 /**
  * Inits the Sentry React Native SDK with automatic instrumentation and wrapped features.
  */
-export function wrap<P extends JSX.IntrinsicAttributes>(
+export function wrap<P = {}>(
```

Comment:
Our lint will complain about this, can you disable it.
```suggestion
// eslint-disable-next-line @typescript-eslint/ban-types
export function wrap<P = {}>(
```